assets/js/lib/model/index.js

```diff
@@ -1,13 +1,21 @@
 export const EXPECT = 'expect';
+export const EXPECT_ENUM = 'expect_enum';
 export const EXPECT_SAME = 'expect_same';
 
+const hostExepectations = [EXPECT, EXPECT_ENUM];
```

Comment:
Just a quick question here, we have multiple files in model directory.  
Like cluster.js, sapSystem.js and index.js

What kind of data should we always put in index.js?

For now, it is mostly cluster details view related.

Reply:
well, that's subjective, as long as it doesn't go out of control having one file is convenient